src/nft/components/bag/Bag.tsx

```diff
@@ -66,7 +66,7 @@ const BagContainer = styled.div<{ raiseZIndex: boolean; isProfilePage: boolean }
   border-radius: 16px;
   box-shadow: ${({ theme }) => theme.shallowShadow};
   z-index: ${({ raiseZIndex, isProfilePage }) =>
-    raiseZIndex ? (isProfilePage ? Z_INDEX.modalOverTooltip : Z_INDEX.modalBackdrop) : 3};
+    raiseZIndex ? (isProfilePage ? Z_INDEX.modalOverTooltip : Z_INDEX.modalBackdrop + 2) : 3};
```

Comment:
q: Do we need the `raiseZIndex` check? It seems like we always want the zindex either `modalOverTooltip` or `modalBackdrop`

Reply:
yeah we actually need the `raiseZIndex` check because otherwise the wallet dropdown will go behind the bag